spec/definitions/GlobalTimelineEvent.yaml

```diff
@@ -0,0 +1,33 @@
+type: object
+description: Global Timeline event.
+readOnly: true
+properties:
+  resourceType:
+    type: string
+    description: Related resource type
+    enum:
+      - transactions
+      - subscriptions
+      - invoices
+      - disputes
```

Comment:
I think we have an event for Payment Card Expired too, so should `payment-cards` be added?